tests/expressions/tests.py

```diff
@@ -591,6 +591,14 @@ def test_pickle_expression(self):
         expr.convert_value  # populate cached property
         self.assertEqual(pickle.loads(pickle.dumps(expr)), expr)
 
+    def test_incorrect_field_expression(self):
+        with self.assertRaisesMessage(FieldError, "Cannot resolve keyword 'nope' into field."):
+            list(Employee.objects.filter(firstname=F('nope')))
```

Comment:
I agree that `test_incorrect_field_expression()` should be in the `BasicExpressionsTests` instead of `ExpressionsNumericTests`, but this change is not related with a fix. Please move this to a separate commit.

Reply:
Agreed.

PS. I take this opportunity to see what happens to comments when rewriting history in PR, haven't done that before.